### PR TITLE
Build instractions

### PR DESCRIPTION
**Pull request by volovyk-s**

### CODE REVIEW

File: docs/building/basic-build.md [hunk 1]
```diff
@@ -2,10 +2,46 @@
...
+cargo build --target wasm32-unknown-unknown --release
+```
 
-<!-- Basic building instructions -->
+You can simpify this command by adding this flags to your `ProjectFolder/.cargo/config.toml`.
 
-<!-- Why we need flags and certain build configuration -->
+```toml
+[target.wasm32-unknown-unknown]
+rustflags = ["-C", "link-arg=-s"]
```
Comment (mikedotexe) [inline docs/building/basic-build.md:44]:
```suggestion
rustflags = ["-C link-arg=-s"]
```

Note that this is something new in near-sdk-rs that Austin discovered is an improvement:
https://github.com/near/near-sdk-rs/pull/543/files#diff-8a51f1a3e37218978813e2fdb533fb9bbb6d687472e2a0a3f8673ba995de3054R2

File: docs/building/basic-build.md [hunk 2]
```diff
@@ -2,10 +2,46 @@
...
+```
 
-TODO
\ No newline at end of file
+You can find an example [here](https://github.com/near/near-sdk-rs/blob/072b47779fcb219f468977bd7e727b4a83d9bd44/examples/fungible-token/.cargo/config.toml).
```
Comment (mikedotexe) [inline docs/building/basic-build.md:47]:
```suggestion
You can find an example [here](https://github.com/near/near-sdk-rs/blob/05e4539a8f3db86dd43b768ee9660dd4c8e7ea5c/examples/fungible-token/.cargo/config.toml).
```

Just updating to the newer version that has slightly different structure to the `rustflags` as shown in the previous comment.

**Review by mikedotexe — APPROVED** [review]
Once these comments are addressed looks great.

File: docs/building/basic-build.md (hunk 1 above)
Comment (austinabell) [reply docs/building/basic-build.md:44]: nah, this is the correct version. See https://github.com/near/near-sdk-rs/commit/05e4539a8f3db86dd43b768ee9660dd4c8e7ea5c.

The version you suggest doesn't work on some archs

Comment (volovyk-s) [reply docs/building/basic-build.md:44]: @mikedotexe yes, I took it from the `near-sdk-rs` repo. Please, resolve/merge if you agree.

**Pull request merged**